csharp/ql/src/Stubs/Stubs.qll

```diff
@@ -368,17 +397,29 @@ private string stubImplementation(Virtualizable c) {
 }
 
 private string stubParameters(Parameterizable p) {
-  result = concat(int i, Parameter param | param = p.getParameter(i) and not param.getType() instanceof ArglistType |
-    stubParameterModifiers(param) + stubClassName(param.getType()) + " " + param.getName() + stubDefaultValue(param), ", " order by i)
+  result = concat(int i, Parameter param |
+      param = p.getParameter(i) and not param.getType() instanceof ArglistType
+    |
+      stubParameterModifiers(param) + stubClassName(param.getType()) + " " + param.getName() +
+          stubDefaultValue(param), ", "
+      order by
+        i
+    )
 }
 
 private string stubParameterModifiers(Parameter p) {
-  if p.isOut() then result = "out "
-  else if p.isRef() then result = "ref "
-  else if p.isParams() then result = "params "
-  else if p.isIn() then result = "" // Only C# 7.1 so ignore
-  else if p.hasExtensionMethodModifier() then result = "this "
-  else result = ""
+  if p.isOut()
+  then result = "out "
+  else
+    if p.isRef()
+    then result = "ref "
+    else
+      if p.isParams()
+      then result = "params "
+      else
+        if p.isIn()
+        then result = "" // Only C# 7.1 so ignore
+        else if p.hasExtensionMethodModifier() then result = "this " else result = ""
```

Comment:
@aschackmull This last line should probably have the `then` and `else` on separate lines to match the rest.

Reply:
Hmm.. that might be doable.  But the current way is more consistent in the sense that each term gets formatted the same way regardless of context.

Reply:
I'll consider this and investigate the impact of such a change.